### PR TITLE
UVC driver release 1.0.4 (IEC-104)

### PR DESCRIPTION
**Pull request by lijunru-hub**
1. support print `format frame based` descriptor
2. support print `frame frame based` descriptor

### CODE REVIEW

**Comment by leeebo** [comment]
@lijunru-hub LGTM! but please add more detailed PR descriptions (eg. the purpose, and specs you referred). 

and please add the scope information in your commit message 

eg. `feat(usb_host_uvc): support printf frame based descriptors`

**Comment by leeebo** [comment]
"USB Video Payload Frame Based v1.5"  https://www.usb.org/document-library/video-class-v15-document-set

**Comment by tore-espressif** [comment]
@lijunru-hub PTAL at the pedantic compiler errors https://github.com/espressif/esp-usb/actions/runs/8795816894/job/24148662774?pr=31#step:4:10619

File: host/class/uvc/usb_host_uvc/src/descriptor.c [hunk 1]
```diff
@@ -730,6 +826,12 @@ static void print_class_specific_desc(const uint8_t *buff)
     case VS_FRAME_MJPEG:
         print_vs_frame_mjpeg_desc(buff);
         break;
+    case VS_FORMAT_FRAME_BASED:
+        print_vs_format_frame_based_desc(buff);
+        break;
+    case VS_FRAME_FRAME_BASED:
+        print_vs_frame_frame_based_desc(buff);
```
Comment (roma-jam) [inline host/class/uvc/usb_host_uvc/src/descriptor.c:833]: Nitpick: 
`print_vs_format_frame_based_desc` and `print_vs_frame_frame_based_desc` - very easy to confuse.
Is there any chance to change the name of  `VS_FORMAT_FRAME_BASED` and `VS_FRAME_FRAME_BASED`?

**Review by roma-jam — COMMENTED** [review]
@lijunru-hub Thanks for the request! 

This suppose to be a release for verison 1.0.4 of uvc host class driver. 

Could you please update the PR description accordingly? (you can refer to https://github.com/espressif/esp-usb/pull/24 or https://github.com/espressif/esp-usb/pull/11)

**Comment by lijunru-hub** [comment]
> @lijunru-hub LGTM! but please add more detailed PR descriptions (eg. the purpose, and specs you referred).
> 
> and please add the scope information in your commit message
> 
> eg. `feat(usb_host_uvc): support printf frame based descriptors`

File: host/class/uvc/usb_host_uvc/src/descriptor.c (hunk 1 above)
Comment (lijunru-hub) [reply host/class/uvc/usb_host_uvc/src/descriptor.c:833]: This is the standard format in the protocol.
image

**Review by roma-jam — APPROVED** [review]
Thanks, 
Changes LGTM.

**Review by leeebo — APPROVED** [review]

**Comment by lijunru-hub** [comment]
@tore-espressif PTAL

**Comment by lijunru-hub** [comment]
@tore-espressif @roma-jam 
Is there still an issue with this PR? If there are no issues, should it be merged?

**Review by tore-espressif — APPROVED** [review]
LGTM! I rebased the PR and we can merge after the CI succeeds

**Pull request merged**